modules/serverless-installing-cli-linux-ibm-power-rpm.adoc

```diff
@@ -0,0 +1,43 @@
+[id="installing-cli-linux-ibm-power-rpm-{context}"]
+== RPM Installation
+
+For Red Hat Enterprise Linux (RHEL), you can install the `kn` CLI as an RPM if you have an active {product-title} subscription on your Red Hat account.
```

Comment:
Please use the attributes where possible for product names like RHEL as well as {product-title} etc. The `common-attributes.adoc` file should already be included in this assembly. Please take a look at the OpenShift docs processes for more info https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata